gulpfile.js

```diff
@@ -56,11 +56,16 @@ var paths = {
     '!bower_components/angular-animate/**/**/*',
     '!bower_components/angular-sanitize/**/**/*',
     '!bower_components/angular-resource/**/**/*',
-    '!bower_components/angular-ui-router/**/**/*'
+    '!bower_components/angular-ui-router/**/**/*',
+    '!bower_components/jquery/**/**/*',
```

Comment:
Because the icons needed are only available as part of the bootstrap(-sass) package, so here I excluded jquery for copy.

Dependency https://github.com/twbs/bootstrap-sass/blob/master/bower.json :38